Source/BSGConnectivity.m

```diff
@@ -41,7 +43,7 @@
  * @return YES if the connectivity change should be reported
  */
 BOOL BSGConnectivityShouldReportChange(SCNetworkReachabilityFlags flags) {
-    #if TARGET_OS_TV || TARGET_OS_IPHONE
```

Comment:
Deliberately not simulator?

Reply:
I can't see any reason why this code shouldn't apply to iOS/tvOS simulators as the `kSCNetworkReachabilityFlagsIsWWAN` flag is only missing on macOS.